projects/fab-speed-dial/src/lib/fab-speed-dial.ts

```diff
@@ -230,6 +253,34 @@ export class EcoFabSpeedDialComponent implements AfterContentInit {
             this.renderer.removeClass(this.elementRef.nativeElement, finalClass);
         }
     }
+
+    private _processOutsideClickState() {
+        if (this.fixed) {
+            return;
+        }
+        if (this._closeOnOutsideClick) {
+            this._setDocumentClickBindings();
+        } else {
+            this._unsetDocumentClickBindings();
+        }
+    }
+
+    private _setDocumentClickBindings() {
+        if (!this._documentListener) {
+            this._documentListener = this.renderer.listen(this.document, 'click', () => {
+                if (!this.fixed && this.open) {
+                    this.open = false;
+                }
+            });
+        }
+    }
+
+    private _unsetDocumentClickBindings() {
+        if (this._documentListener) {
+            this._documentListener();
```

Comment:
Probably missing a `this._documentListener = null` either here or in the listener itself... depends if we want to remove the listener after each click or not...

Reply:
Indeed it was missing. Anyway I also did change the listener registering/unregistering, it is registered when actions are visible and unregistered when hidden.